picard/config.py

```diff
@@ -451,7 +451,8 @@ def convert(self, value):
 profiles = None
 
 
-def setup_config(app, filename=None):
+def setup_config(filename=None):
+    app = QtCore.QCoreApplication.instance()
```

Comment:
I think this is one place where we could better keep the parameter. This function is straight forward be supposed to be called from the instantiated app, where this variable is available.

Reply:
See last changes in https://github.com/metabrainz/picard/pull/2424/commits/afa49dae4fab1e49cd3b51c4825f62e293a716a0